### DISCUSSION TITLE
Knowledge with Chat - Beta Signup

### DISCUSSION THREAD

**Discussion by RobRoyce** [KnowledgeCanvas/knowledge#106] [Announcements]
For anyone interested in beta testing the new Chat feature, please [fill out the signup form](https://forms.gle/aJPe7FhwypRbZvRa6). 

I'm making some final adjustments to the UI and doing some last minute testing before getting the beta release out. The stable release will follow sometime after.

**Comment by rich-martin** [comment]
Hey Rob, is this something that’s coming imminently, or just in the near future? I’ve got a big project it would be perfect for!
